pom.xml

```diff
@@ -89,7 +93,135 @@
+		<file-management.version>3.0.0</file-management.version>
+		<maven.version>3.3.9</maven.version>
+		<maven-plugin-annotations.version>3.6.0</maven-plugin-annotations.version>
+		<maven-plugin-testing-harness.version>3.3.0</maven-plugin-testing-harness.version>
+		<plantuml-maven-plugin-version>1.2</plantuml-maven-plugin-version>
+		<plantuml-version>8059</plantuml-version>
+		<xml-maven-plugin-version>1.0.2</xml-maven-plugin-version>
+		<fix.orchestra.version>1.5.4</fix.orchestra.version>
+		<orchestra.repository.version>1.6.8</orchestra.repository.version>
+		<docgen.version>1.6.8</docgen.version>
+		<jaxb.version>2.3.3</jaxb.version>
+		<apache.mina.version>2.1.5</apache.mina.version>
+		<commons.io.version>2.11.0</commons.io.version>
+		<guava.version>31.0.1-jre</guava.version>
+		<orchestra.file>OrchestraFIXLatest.xml</orchestra.file>
+   </properties>
+
+   <dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>io.fixprotocol.orchestrations</groupId>
+				<artifactId>fix-standard</artifactId>
+				<version>${fix.orchestra.version}</version>
+			</dependency>
+			<dependency>
+				<groupId>io.fixprotocol.orchestra</groupId>
+				<artifactId>repository</artifactId>
+				<version>${orchestra.repository.version}</version>
+			</dependency>
+			<dependency>
+				<groupId>org.junit.jupiter</groupId>
+				<artifactId>junit-jupiter-engine</artifactId>
+				<version>${junit.jupiter.version}</version>
+				<scope>test</scope>
+			</dependency>
+			<dependency>
+				<groupId>junit</groupId>
+				<artifactId>junit</artifactId>
+				<version>${junit.version}</version>
+				<scope>test</scope>
+			</dependency>			
+			<dependency>
+				<groupId>org.junit.jupiter</groupId>
+				<artifactId>junit-jupiter-params</artifactId>
+				<version>${junit.jupiter.version}</version>
+				<scope>test</scope>
+			</dependency>
+			<dependency>
+				<groupId>org.mockito</groupId>
+				<artifactId>mockito-core</artifactId>
+				<version>${mockito-core.version}</version>
+				<scope>test</scope>
+			</dependency>
+			<dependency>
+				<groupId>org.hamcrest</groupId>
+				<artifactId>hamcrest-all</artifactId>
+				<version>${hamcrest-all.version}</version>
+				<scope>test</scope>
+			</dependency>			
+			<dependency>
+				<groupId>jakarta.xml.bind</groupId>
+				<artifactId>jakarta.xml.bind-api</artifactId>
+				<version>${jaxb.version}</version>
+			</dependency>
+			<dependency>
+				<groupId>com.sun.xml.bind</groupId>
+				<artifactId>jaxb-impl</artifactId>
+				<scope>runtime</scope>
+				<version>${jaxb.version}</version>
+			</dependency>
+			<dependency>
+				<groupId>org.apache.mina</groupId>
+				<artifactId>mina-core</artifactId>
+				<version>${apache.mina.version}</version>
+			</dependency>
+			<dependency>
+			    <groupId>commons-io</groupId>
+			    <artifactId>commons-io</artifactId>
+			    <version>${commons.io.version}</version>
+			</dependency>
+			<dependency>
+			  <groupId>com.google.guava</groupId>
```

Comment:
What is guava used for?

Reply:
Used in Maven plugins.
<img width="691" alt="image" src="https://user-images.githubusercontent.com/29655222/207682706-2e10c200-0ba4-4939-aac7-1e67cbca8aea.png">

Reply:
OK, but isn't this a transitive dependency then? Does it need to be declared in the POM?

Reply:
It does need to be declared when you coerce (manage)  a transitive dependency to use a later version, in this case 31. 
I looked through the code and in the _full restructured_ code a guava feature is used in the project's mvn plugin source code. 
In this PR I will test to see if mvn plugins work with version 18.

Reply:
@chrjohn - PR updated.